apps/site/sentry.server.config.ts

```diff
@@ -1,7 +1 @@
-// Context: https://docs.sentry.io/platforms/javascript/guides/nextjs/
-
-import * as Sentry from "@sentry/nextjs";
-
-import { sentryConfig } from "./sentry.client.config";
-
-Sentry.init(sentryConfig);
```

Comment:
I guess we don't need `Sentry.init` because `sentry.client.config` has this side effect too.